virtualization/windowscontainers/quick-start/quick-start-windows-server.md

```diff
@@ -50,45 +70,66 @@ When PowerShell asks you whether to trust the package source 'DockerDefault', ty
 ```
 Restart-Computer -Force
 ```
+Please go to section 4 now, if you are happy with the Community Edition.
```

Comment:
If the goal is to combine production Docker EE install instructions with Insider Docker EE instructions then only one or the other will be used (Docker EE vs Docker EE Preview). This sentence should be removed.